pkg/manifests/manifests.go

```diff
@@ -1455,6 +1459,14 @@ func (f *Factory) PrometheusK8s(grpcTLS *v1.Secret, trustedCABundleCM *v1.Config
 
 	if len(f.config.ClusterMonitoringConfiguration.PrometheusK8sConfig.RemoteWrite) > 0 {
 		p.Spec.RemoteWrite = addRemoteWriteConfigs(clusterID, p.Spec.RemoteWrite, f.config.ClusterMonitoringConfiguration.PrometheusK8sConfig.RemoteWrite...)
+
+		// Explicitly warn that we are ignoring the `SendExemplars` configuration for in-cluster scenarios.
+		for _, rws := range f.config.ClusterMonitoringConfiguration.PrometheusK8sConfig.RemoteWrite {
+			if rws.SendExemplars != nil && *rws.SendExemplars {
+				klog.Warningln("Enabling `SendExemplars` in the in-cluster Prometheus configuration has no affect, as it is only supported for UWM configurations.")
```

Comment:
I'm not sure we need a warning log. IMHO it would be lost in the existing output + there's no harm being done by having the flag enabled here.

Reply:
@danielmellado Ah, it seems I missed your comment (I reloaded this page after >1h). Do you suppose we replace `Warningln` with a `V(x)` level call?